GameOfLifeApp/Models/GameField.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace GameOfLifeApp
+{
+    public class GameField : IGameField
+    {        
+        /// <summary>
+        /// Holds user data of array size.    
+        /// </summary>        
+        public int CountOfRows { get; set; }
+
+        /// <summary>
+        /// Holds user data of array size. 
+        /// </summary>        
+        public int CountOfColumns { get; set; }
+
+        /// <summary>
+        /// Currently active game field.
+        /// </summary>        
+        public bool[,] CurrentField { get; set; }
+
+        public int CountIteration { get; set; }
```

Comment:
if you have summary for the method or property it should be the same in the interfaces, so please update all the interfaces with method summaries.

Reply:
Done.